.github/workflows/ci-benchmark.yml

```diff
@@ -16,46 +16,67 @@ name: CI Benchmark
 
 on:
   push:
+    branches:
+    - master
     paths-ignore:
     - 'docs/**'
     - '.github/workflows/ci-docs.yml'
     - 'requirements.txt'
     - 'docs.Dockerfile'
     - '*.md'
+  pull_request_target:
```

Comment:
Unfortunately this new type does not work either (but due to a different reason). https://github.com/li-boxuan/janusgraph/runs/6824416682?check_suite_focus=true#step:6:11 Apparently with this event, we cannot download previous benchmark data from the cache.